packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/iOS/signin.md

```diff
@@ -28,23 +12,11 @@ oktaOidc.signInWithBrowser(from: self) { stateManager, error in
     // stateManager.refreshToken
 }
 ```
-You are responsible for storing the state manager. You can securely store the `OktaOidcStateManager` instance in the iOS keychain:
+
+You are responsible for persisting the state manager. You can securely store the `OktaOidcStateManager` instance in the iOS keychain:
 
 ```swift
 authStateManager.writeToSecureStorage()
 let config = OktaOidcConfig(/* plist */)
```

Comment:
@IldarAbdullin-okta Is this the same config that the developer created earlier?

Reply:
Does it make sense to move these two lines (retrieving the stateManager from storage) into the next step, where the stateManager is needed again?

Reply:
Ok, let's do the following:
- save stateManager in callback if `signInWithBrowser ` which is the right place for that
- then in the next code snippet show how to retrieve stateManager